Assets node_modules folder breaks Docker build on Windows

# Steps to Reproduce
It's important to note that I've only been able to reproduce this issue on Windows, I don't seem to run into these issues on Mac/Linux.

1. Perform a fresh `git clone` of bors-ng
2. `cd .\assets\` and run `npm i`
3. `cd ..` and run `docker build -t bors-ng:repro .`

Following these steps produces the following build failure:

```
 > [builder  8/11] RUN cd /src/ && npm install && npm run deploy:
#17 0.807 npm notice created a lockfile as package-lock.json. You should commit this file.
#17 0.815 npm WARN bors-ng-assets@0.0.0 No repository field.
#17 0.822 npm WARN bors-ng-assets@0.0.0 No license field.
#17 0.823
#17 1.028 up to date in 0.297s
#17 1.028 found 0 vulnerabilities
#17 1.028
#17 1.225
#17 1.225 > bors-ng-assets@0.0.0 deploy /src
#17 1.225 > cd assets && NODE_ENV=dev npm install && npm run deploy
#17 1.225
#17 4.631 npm WARN checkPermissions Missing write access to /src/assets/node_modules/phoenix
#17 4.632 npm WARN checkPermissions Missing write access to /src/assets/node_modules/phoenix_html
#17 4.895 npm WARN assets No description
#17 4.903 npm WARN assets license should be a valid SPDX license expression
#17 4.904
#17 4.915 npm ERR! code ENOENT
#17 4.915 npm ERR! syscall access
#17 4.916 npm ERR! path /src/assets/node_modules/phoenix
#17 4.916 npm ERR! errno -2
#17 4.917 npm ERR! enoent ENOENT: no such file or directory, access '/src/assets/node_modules/phoenix'
#17 4.917 npm ERR! enoent This is related to npm not being able to find a file.
#17 4.917 npm ERR! enoent
#17 4.930
#17 4.930 npm ERR! A complete log of this run can be found in:
#17 4.930 npm ERR!     /root/.npm/_logs/2021-03-25T00_21_59_212Z-debug.log
#17 4.944 npm ERR! code ELIFECYCLE
#17 4.944 npm ERR! errno 254
#17 4.950 npm ERR! bors-ng-assets@0.0.0 deploy: `cd assets && NODE_ENV=dev npm install && npm run deploy`
#17 4.950 npm ERR! Exit status 254
#17 4.950 npm ERR!
#17 4.950 npm ERR! Failed at the bors-ng-assets@0.0.0 deploy script.
#17 4.950 npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
#17 4.959 npm WARN Local package.json exists, but node_modules missing, did you mean to install?
#17 4.959
#17 4.959 npm ERR! A complete log of this run can be found in:
#17 4.960 npm ERR!     /root/.npm/_logs/2021-03-25T00_21_59_244Z-debug.log
------
executor failed running [/bin/sh -c cd /src/ && npm install && npm run deploy]: exit code: 254
```

I believe the cause to be file permissions errors caused by the `assets/node_modules` directory being copied into the Docker context.

# Solution
I'm not aware of any reason for copying the `node_modules` directory into the Docker context, and removing it (by adding it to the .dockerignore) fixes the build issues I've been encountering. I'd like make a PR adding the following directories and files to the .dockerignore:

- /node_modules
- /assets/node_modules
- /doc
- erl_crash.dump
- **/_build
- /db
- **/deps
- /*.ez
- /.deliver/releases/
- /.idea
- /*.iml
- /projectFilesBackup
- /projectFilesBackup1

All of these are from the .gitignore, with the idea that if it isn't in the repository, it shouldn't be required to build Bors.

--- a/.dockerignore
+++ b/.dockerignore
@@ -1,2 +1,15 @@
 _build
 deps
+/node_modules
+/assets/node_modules
+/doc
+erl_crash.dump
+**/_build
+/db
+**/deps
+/*.ez
+/.deliver/releases/
+/.idea
+/*.iml
+/projectFilesBackup
+/projectFilesBackup1

